cmd/libs/go2idl/parser/parse.go

```diff
@@ -495,3 +504,14 @@ func (b *Builder) walkType(u types.Universe, useName *types.Name, in tc.Type) *t
 		return out
 	}
 }
+
+func (b *Builder) addFunc(u types.Universe, useName *types.Name, in *tc.Func) *types.Type {
+	name := tcFuncNameToName(in.String())
+	if useName != nil {
+		name = *useName
+	}
+	out := u.Get(name)
```

Comment:
@lavalamp we verified that Golang does not allow global variable share the same name with a function yesterday, but Go allows a method to share the same name with a global variable, so we still need a patch here.